### DISCUSSION TITLE
파일명을 변경하고 싶어요.

### DISCUSSION THREAD

**Discussion by KUBONKIL** [bmrdevteam/Altsis#292] [질문과 답변]
![image](https://github.com/bmrdevteam/school-information-system/assets/143776178/c0e83f57-7d7d-45ea-860f-28e6f56552cd)
이름바꾸기 기능을 알려주세요.

**Comment by devgoodway** [comment]
파일명은 수정 페이지에 들어가셔서 상단 제목 부분을 클릭하시고 이름을 수정하시면 파일 이름이 수정됩니다!

**Reply by KUBONKIL** [reply]
확인 되었습니다. 감사합니다.
